.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         # To solve: increment the CACHE_VERSION environment variable in the CircleCI Project settings.
         # Example: if the current CACHE_VERSION is 'xxxx2' (this value isn't sensitive, but CircleCI masks it)
         # then it means the current version is 'v2', so the next should be 'v3'.
-      - image: cimg/rust:1.64.0
+      - image: cimg/rust:1.70.0
```

Comment:
I think the server is on [1.68](https://github.com/launchdarkly/rust-server-sdk/blob/main/.github/workflows/ci.yml#L23) - is 1.70 necessary?

Reply:
Yes, unfortunately.

Reply:
```error: failed to compile `cargo-all-features v1.10.0`, intermediate artifacts can be found at `/tmp/cargo-installwxwoPC````

We could move away from this tool. That might lower this back down to 1.68.